plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -321,6 +356,7 @@ class CodeWhispererService(private val cs: CoroutineScope) : Disposable {
                     val responseContext = ResponseContext(sessionId)
 
                     CodeWhispererTelemetryService.getInstance().sendServiceInvocationEvent(
+                        currentJobId,
```

Comment:
let's say there're 2 sessions with 4 triggers and 3 triggers respectively, are we sending something like jobid = [1, 2, 3, 4] for the first session and [5, 6, 7] the 2nd session?

Reply:
yes, jobId only increments. It's a local field that only helps logging and not sent to telemetry.